README.md

```diff
@@ -1,3 +1,7 @@
+# Attention
+
+Can only use `npm install` to install dependences
+
```

Comment:
I think it is not appropriate to add this here since `README` is showed for users rather than developers.
If you really want to tell the later contributors, I think the `Contribute` section is a better place.